config/config.go

```diff
@@ -0,0 +1,33 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/tkanos/gonfig"
+)
+
+type Configuration struct {
+	DbHostname string `json:"DB_HOSTNAME"`
+	DbUser     string `json:"DB_USER"`
+	DbPassword string `json:"DB_PASSWORD"`
+	DbName     string `json:"DB_NAME"`
```

Comment:
By Golang convention, this should be rewriten as following:
```suggestion
	DBHostname string `json:"DB_HOSTNAME"`
	DBUser     string `json:"DB_USER"`
	DBPassword string `json:"DB_PASSWORD"`
	DBName     string `json:"DB_NAME"`
```